src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -22,8 +22,8 @@ private void TestWindowApisUnix()
             // This tests the hardcoded implementations of these APIs on Unix.
             using (Process p = Process.GetCurrentProcess())
             {
-                Assert.True(p.Responding);
-                Assert.Equal(string.Empty, p.MainWindowTitle);
+                Assert.Throws<PlatformNotSupportedException>(() => p.Responding);
```

Comment:
ok, let's sum this up:
<table border="2" cellspacing="0" cellpadding="6" rules="groups" frame="hsides">


<colgroup>
<col  class="org-left" />

<col  class="org-left" />

<col  class="org-left" />

<col  class="org-left" />
</colgroup>
<thead>
<tr>
<th scope="col" class="org-left">Member</th>
<th scope="col" class="org-left">UAP</th>
<th scope="col" class="org-left">Linux</th>
<th scope="col" class="org-left">IMO: both</th>
</tr>
</thead>

<tbody>
<tr>
<td class="org-left">IntPtr MainWindowHandle</td>
<td class="org-left">PNSE</td>
<td class="org-left">null</td>
<td class="org-left">PNSE</td>
</tr>


<tr>
<td class="org-left">bool CloseMainWindow()</td>
<td class="org-left">PNSE</td>
<td class="org-left">false</td>
<td class="org-left">PNSE</td>
</tr>


<tr>
<td class="org-left">string MainWindowTitle</td>
<td class="org-left">PNSE</td>
<td class="org-left">string.Empty</td>
<td class="org-left">PNSE</td>
</tr>


<tr>
<td class="org-left">bool Responding</td>
<td class="org-left">PNSE</td>
<td class="org-left">true</td>
<td class="org-left">PNSE</td>
</tr>


<tr>
<td class="org-left">bool WaitForInput</td>
<td class="org-left">PNSE</td>
<td class="org-left">false</td>
<td class="org-left">PNSE</td>
</tr>
</tbody>
</table>